### DISCUSSION TITLE
Custom Bone Shape Collision

### DISCUSSION THREAD

**Discussion by radpas1213** [shteeve3d/blender-wiggle-2#11] [Ideas]
Add the ability to have custom collisions for the bones, a good example of usage is if you want a chain to have its own collision for each chain so it behaves more realistically.

I also suggested this because of a project I'm doing with ammo belts, that should be a goof use for it too.

**Comment by shteeve3d** [comment]
Right now collisions are approximated using a simple point radius values around the head and/or tail of the bone. Its unfortunately not a full rigid body solver, which would likely be necessary for the effect I think you're going for.

At that point it might be better to leverage blender's dedicated rigid body physics system for the task, and then constrain your bones to the desired meshes if you want to convert it to bone animation.
